template/default.tmpl

```diff
@@ -190,3 +190,6 @@ Alerts Resolved:
 {{ end }}
 {{ end }}
 {{ define "pushover.default.url" }}{{ template "__alertmanagerURL" . }}{{ end }}
+
+{{ define "jira.default.summary" }}{{ template "__subject" . }}{{ end }}
```

Comment:
i'll show you the test ticket. the default __subject is quite clear that its an alert, although we'll probably configure ours to be a little more readable.